org.darktable.Darktable.json

```diff
@@ -13,12 +13,11 @@
         "--socket=x11",
         "--filesystem=host",
         "--talk-name=org.freedesktop.secrets",
-        /* Needed for dconf to work */
-        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
-        "--filesystem=xdg-run/gvfs",
-        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
```

Comment:
But it is missing the migration as explained at:

https://blogs.gnome.org/mclasen/2019/07/12/settings-in-a-sandbox-world/

Reply:
Does darktable even use dconf?
Using the dconf editor, I couldn't find anything from darktable.

Reply:
To be safe, I can still add the line, if you want me to.

Reply:
I believe it should.

Reply:
Done.